static/js/components/AssignmentForm.jsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { useForm, Controller } from 'react-hook-form';
+import Select from "@material-ui/core/Select";
+import InputLabel from "@material-ui/core/InputLabel";
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import FormControl from "@material-ui/core/FormControl";
+import MenuItem from "@material-ui/core/MenuItem";
+import { makeStyles } from "@material-ui/core/styles";
+import * as Actions from '../ducks/source';
+
+import 'react-datepicker/dist/react-datepicker-cssmodules.css';
+
+export function observingRunTitle(observingRun, instrumentList, telescopeList, groups) {
+  const { instrument_id } = observingRun;
+  const instrument = instrumentList?.filter((i) => i.id === instrument_id)[0];
+
+  const telescope_id = instrument?.telescope_id;
+  const telescope = telescopeList?.filter((t) => t.id === telescope_id)[0];
+
+  const group = groups?.filter((g) => g.id === observingRun.group_id)[0];
+
+  return `${observingRun?.calendar_date} ${instrument?.name}/${telescope?.nickname} (PI: ${observingRun?.pi} ${group?.name})`;
+}
+
+const AssignmentForm = ({ obj_id, observingRunList }) => {
+  const dispatch = useDispatch();
+  const { instrumentList } = useSelector((state) => state.instruments);
+  const { telescopeList } = useSelector((state) => state.telescopes);
+  const groups = useSelector((state) => state.groups.all);
+
+  const upcomingRuns = observingRunList.filter((observingrun) => (
+    Date.parse(observingrun.ephemeris.sunrise_utc) >= Date.now()
```

Comment:
> It is not recommended to use Date.parse as until ES5, parsing of strings was entirely implementation dependent. There are still many differences in how different hosts parse date strings, therefore date strings should be manually parsed (a library can help if many different formats are to be accommodated).

Can we use dayJS here? https://day.js.org/docs/en/parse/utc

Reply:
Replaced with:

    dayjs.utc(observingrun.ephemeris.sunrise_utc) >= dayjs()